Application/EdFi.Ods.Admin/Controllers/AccountController.cs

```diff
@@ -135,16 +127,16 @@ public ActionResult ResetPassword(PasswordResetModel model)
         }
 
         [HttpGet]
-        public async Task<ActionResult> ActivateAccount(string marker)
+        public ActionResult ActivateAccount(string email, string marker)
         {
-            var validation = await _passwordService.ConfirmAccountAsync(marker);
+            var validation = _passwordService.ConfirmAccount(email,marker);
 
             if (validation.Success)
             {
                 return View(
                     new PasswordResetViewModel
                     {
-                        Marker = marker, UserName = validation.UserName
+                        Marker = marker, Email = validation.UserName
```

Comment:
Should this UserName property on ConfirmationResult also be updated to be "Email"?